test/e2e/node/pod_resize.go

```diff
@@ -44,21 +44,19 @@ import (
 )
 
 const (
-	CgroupCPUPeriod string = "/sys/fs/cgroup/cpu/cpu.cfs_period_us"
-	CgroupCPUShares string = "/sys/fs/cgroup/cpu/cpu.shares"
-	CgroupCPUQuota  string = "/sys/fs/cgroup/cpu/cpu.cfs_quota_us"
-	CgroupMemLimit  string = "/sys/fs/cgroup/memory/memory.limit_in_bytes"
+	CgroupCPUPeriod    string = "/sys/fs/cgroup/cpu/cpu.cfs_period_us"
```

Comment:
Please sync to my latest code. Then this will not be part of the PR since I already merged it.